site/docs/voice/quickStart.mdx

```diff
@@ -32,8 +32,8 @@ Bandwidth will expect BXML in the HTTP response from either of these URLs to con
 These URLs can be the same or different, depending on if you would like inbound and outbound calls to be handled differently.
 
 
-### Call Status Callbacks
-Bandwidth will also send call status updates to the configured `CallStatusCallbackUrl`. These webhooks are useful for error checking and include events such as _rejected_ and _hung up_.
+### Disconnect Callbacks
```

Comment:
Should we have a paragraph for initiate?

Reply:
We kind of do above this but I could be convinced to roll both of these into a single paragraph

Reply:
Yeah a single paragraph would be fine, imo